t/01-temp-files.t

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/perl
+# This program is open source, licensed under the PostgreSQL License.
+# For license terms, see the LICENSE file.
+#
+# Copyright (C) 2012-2022: Open PostgreSQL Monitoring Development Group
+
+use strict;
+use warnings;
+
+use lib 't/lib';
+use pgNode;
+use pgSession;
+use File::Path qw(rmtree);
```

Comment:
I can't find a place where you use `rmtree`. You probably can remove this line.
```suggestion
```

Reply:
Indeed, the cluster is scratched after each **successful** script run. It is kept on error so we can investigate the error.